internal/bot/verify_requests.go

```diff
@@ -0,0 +1,40 @@
+package bot
+
+import (
+	"bytes"
+	"context"
+	"hellper/internal/config"
+	"hellper/internal/log"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/slack-go/slack"
+)
+
+func VerifyRequests(r *http.Request, w http.ResponseWriter, f http.Handler) {
```

Comment:
I think this function is not working, because if the request fails in the verification, we just log it and continue execution. I think we must return the request with an error.